examples/dynamic-imports/webpack.config.js

```diff
@@ -0,0 +1,15 @@
+const path = require('path')
+const { getWebpackConfig } = require('@financial-times/dotcom-build-webpack-config')
+const js = require('@financial-times/dotcom-build-js')
+
+module.exports = getWebpackConfig(
+  {
+    entry: {
+      scripts: './client/main.js'
+    },
+    output: {
+      publicPath: '/public/'
```

Comment:
Did you mean `path.resolve('public')`?

Reply:
this is `publicPath`, it's for a URL not a file path